### PR TITLE
Update quick marc converter to convert indicator's empty spaces

### PR DESCRIPTION
**Pull request by viacheslavpoliakov**
### Purpose
Convert empty spaces values to '#'

### Approach
Add replacement for empty spaces inside quick marc to marc record converter fro indicators

### Changes Checklist
- [ ] **API Changes**: Document any API paths, methods, request or response bodies changed, added, or removed.
- [ ] **Database Schema Changes**: Indicate any database schema changes and their impact. Confirm that migration scripts were created.
- [ ] **Interface Version Changes**: Indicate any changes to interface versions.
- [ ] **Interface Dependencies**: Document added or removed dependencies.
- [ ] **Permissions**: Document any changes to permissions.
- [ ] **Logging**: Confirm that logging is appropriately handled.
- [x] **Unit Testing**: Confirm that changed classes were covered by unit tests.
- [ ] **Integration Testing**: Confirm that changed logic was covered by integration tests.
- [ ] **Manual Testing**: Confirm that changes were tested on local or dev environment.
- [x] **NEWS**: Confirm that the NEWS file is updated with relevant information about the changes made in this pull request.

### Related Issues
[MODQM-443](https://issues.folio.org/browse/MODQM-443)

### CODE REVIEW

File: src/main/java/org/folio/qm/converter/MarcQmToMarcRecordConverter.java [hunk 1]
```diff
@@ -27,6 +27,9 @@
 @RequiredArgsConstructor
 public class MarcQmToMarcRecordConverter implements Converter<BaseMarcRecord, MarcRecord> {
 
+  private static final char EMPTY_SPACE_VALUE = 32;
+  private static final char MARC_INDICATOR_EMPTY_VALUE = '#';
+
```
Comment (psmagin) [inline src/main/java/org/folio/qm/converter/MarcQmToMarcRecordConverter.java:32]:
```suggestion
  private static final char EMPTY_SPACE_VALUE = ' ';
  private static final char MARC_INDICATOR_EMPTY_VALUE = '#';
```
To be consistent.

File: NEWS.md [hunk 2]
```diff
@@ -9,6 +9,7 @@
 ### Features
 * Implement endpoint to validate record based on MARC specification ([MODQM-433](https://issues.folio.org/browse/MODQM-433))
 * Update cached specification on kafka event ([MODQM-436](https://issues.folio.org/browse/MODQM-436))
+* Update QM to MarcRecord converter ([MODQM-443](https://issues.folio.org/browse/MODQM-443))
```
Comment (psmagin) [inline NEWS.md:12]: Please change it to smth more specific

**Review by psmagin — APPROVED** [review]

File: src/main/java/org/folio/qm/converter/MarcQmToMarcRecordConverter.java (hunk 1 above)
Comment (viacheslavpoliakov) [reply src/main/java/org/folio/qm/converter/MarcQmToMarcRecordConverter.java]: updated

File: NEWS.md (hunk 2 above)
Comment (viacheslavpoliakov) [reply NEWS.md]: updated

**Review by mukhiddin-yusuf — APPROVED** [review]

**Review by SvitlanaKovalova1 — APPROVED** [review]

**Pull request merged**